src/EmployerAccounts/SFA.DAS.Approvals.FakeApis/ReservationsApiBuilder.cs

```diff
@@ -0,0 +1,70 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using AutoFixture;
+using SFA.DAS.EmployerAccounts.InnerApi.Responses;
+using WireMock.RequestBuilders;
+using WireMock.ResponseBuilders;
+using WireMock.Server;
+
+namespace SFA.DAS.EmployerAccounts.FakeApis
+{
+    public class ReservationsApiBuilder
+    {
+        private readonly WireMockServer _server;
+        private readonly Fixture _fixture;
```

Comment:
created this class level Fixture and then not used it, instead opting for `new Fixture()` in the `WithReservations()` method

Reply:
done

Reply:
done